.github/workflows/minikube-k8s-test.yml

```diff
@@ -70,3 +70,23 @@ jobs:
           fi
           echo "logs from pod to make sure:"
           cat pod.log
+
+  cypress-tests:
+    name: Run Cypress Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Comment:
Can we use 21 here?

Reply:
Ok I'll update that, are there any other changes along with this